code/lib/types/src/modules/api.ts

```diff
@@ -127,10 +127,7 @@ export interface API_Notification {
     headline: string;
     subHeadline?: string | any;
   };
-  icon?: {
-    name: string;
-    color?: string;
-  };
```

Comment:
This is a breaking change.

Reply:
I added both types to make it backward compatible,